src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -138,13 +138,7 @@ public static IResourceBuilder<KeycloakResource> WithDataBindMount(this IResourc
     {
         ArgumentNullException.ThrowIfNull(importDirectory);
 
-        if (!Directory.Exists(importDirectory))
```

Comment:
Should we add a test here now that specifies a non existing directory and make sure we throw the right exception?

Reply:
Added.